Server/rainbow-server/Sources/Application/Routes/ScoreRoutes.swift

```diff
@@ -76,7 +76,9 @@ func updateEntry(id: String,newEntry: ScoreEntry, completion: @escaping (ScoreEn
                 /// the user finished a game
                 /// call push notification service
                 Log.info("Sending push notification")
-                pushNotification?.sendNotification(scoreEntry: entry!)
+                DispatchQueue.global(qos: .background).async {
```

Comment:
Let's avoid force unwrapping the main object. We can do this with:

```swift
guard let entry = entry else {
    return // or do whatever we want to do if this doesn't exist
}
pushNotification?.sendNotification(scoreEntry: entry)
```

Also worth pointing out that we don't need to offload this to a background `DispatchQueue` since this is on a server, and we can make use of all the threads we want 😄

Reply:
I was trying to run it as background because even if this fails it doesn't impact the regular flow